blog/2023-10-25-image-editor.md

```diff
@@ -197,9 +208,9 @@ and then find our `H` by adding two large-triangle sides:
   </tbody>
 </table>
 
-So, now we have our desired `H`! One of our scaling factors is `H/h`. Another
-scaling factor of ours will be `W/w`, calculated similarly.  
-The full solution will look as following:
+So, now we have our desired `H`! One of our scaling factors is `H/h`. Our other
+scaling factor is `W/w`, which follows a similar process. Below is a rough
+solution.
```

Comment:
It makes it sound like "below is a calculation of the `W/w` part", while it's not, it's just a full solution. Also - it's not a rough solution, it's a proper solution.